### PR TITLE
New AccessibilityInfo component for UnitView

### PR DESCRIPTION
**Pull request by Dremars**
New AccessibilityInfo component that lists accessibility problems related to unit and specific shortcomings based on user's accessibility settings.

### CODE REVIEW

**Comment by Dremars** [comment]
[Esteettömyystiedot](https://trello.com/c/FiOlBVl1/38-esteett%C3%B6myystiedot)

**Comment by Dremars** [comment]
[Esteettömyystiedot](https://trello.com/c/FiOlBVl1/38-esteett%C3%B6myystiedot)

File: src/views/UnitView/components/AccessibilityInfo/AccessibilityInfo.js [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+                const title = getLocaleText(group);
+
+                if (Object.prototype.hasOwnProperty.call(sentences, key)) {
+                  groupSentences = sentences[key];
+                }
+
+                if (groupSentences && groupSentences.length > 1) {
+                  return (
+                    <div key={title}>
+                      <Typography component={heading} variant="subtitle2" align="left">
```
Comment (mikkoisa) [inline src/views/UnitView/components/AccessibilityInfo/AccessibilityInfo.js]: According to the layout designs this variant should be "subtitle2"

File: src/views/UnitView/components/AccessibilityInfo/AccessibilityInfo.js [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+                        {title}
+                      </Typography>
+                      <ul>
+                        {
+                          groupSentences.map((sentence) => {
+                            const text = getLocaleText(sentence);
+                            return (
+                              <li key={text}>
+                                <Typography component="p" variant="body1" align="left">
```
Comment (mikkoisa) [inline src/views/UnitView/components/AccessibilityInfo/AccessibilityInfo.js]: And this should be "body2"

**Comment by mikkoisa** [comment]
There should also be a bit more padding between the bullet and the text (check the layout designs)

Näyttökuva 2019-6-19 kello 11 34 03

The color of the text is also a bit lighter on the designs. However some other body2 elements on the app have darker color, so we cannot just change the color of body2.

**Pull request merged**